docs/index.rst

```diff
@@ -27,9 +27,8 @@ Girder: a data management platform
 What is Girder?
 ---------------
 
-Girder is a free and open source web-based **data management platform** developed by
-`Kitware <http://www.kitware.com/>`_ as part of the
-`Resonant data and analytics ecosystem <http://resonant.kitware.com/>`_. What does
+Girder is a free and open source web-based **data management platform**, developed by
+`Kitware <http://www.kitware.com/>`_. What does
 that mean? Girder is both a standalone application and a platform for building new web
 services. It's meant to enable quick and easy construction of web applications
 that have some or all of the following requirements:
```

Comment:
I think this sentence is awkward after removing the 'part of Resonant'. Also I am not sure we want to suggest using it for new applications here.

Reply:
@marySalvi That's a reasonable point, but I think it's out of scope for this change.

I'd be fine if you want to change the language in another PR with a different scope.